### ISSUE TITLE
Mismatching childNodes vs. VNodes

### ISSUE THREAD

**Issue by mlsen** [nuxt-community/apollo-module#352]
### Version

[4.0.1-rc.3](https://github.com/apollo-module/releases/tag/4.0.1-rc.3)

### Reproduction link

[https://github.com/mlsen/nuxt-apollo-report](https://github.com/mlsen/nuxt-apollo-report)

### Steps to reproduce

1. yarn install
2. yarn dev
3. Visit the index page and have a look at the console

### What is expected ?

No errors and data being loaded on the server (SSR)

### What is actually happening?

It's giving me a warning

`Mismatching childNodes vs. VNodes`

and an error 

`The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside <p>, or missing <tbody>. Bailing hydration and performing full client-side render.`

The data is then fetched on the client again.

This only happens using the "apollo" object in my component, asyncData works fine.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/apollo-module/issues/c333">#c333</a>)</em></sub></div>

**Comment by daspete** [comment]
hi there... we had the same issue, but we had a custom "globalName" value in the nuxt config. After commenting out this key in the nuxt config, we haven't seen the hydration error again... even with the newest version of nuxt-apollo

**Comment by kukosek** [comment]
same issue here :/

**Comment by kukosek** [comment]
So downgrading to the version you mentioned wasn't the only thing i had to do. I also had disabled caching in my query. So when I disabled it, it works.


So now that I enabled cache I also have an issue that when I change the query parameters, and then I change back to the previous parameters, it tries to load from the cache, but the v-for I use isn't really updated or something. Idk if the caching even works? But that is for another issue i think.
EDIT: nevermind, i solved it by calling `this.$apollo.queries.myQuery.refresh()` after updating the variables
